### PR TITLE
log/channels: Formatting. Also log channel_create_internal_rpc()

### PR DESCRIPTION
**Pull request by justinmk**
Some minor changes from another branch.

### CODE REVIEW

File: src/nvim/channel.c [hunk 1]
```diff
@@ -422,7 +424,7 @@ uint64_t channel_connect(bool tcp, const char *address,
     rstream_start(&channel->stream.socket, on_socket_output, channel);
   }
 
-  channel_create_event(channel, NULL);
+  channel_create_event(channel, address);
```
Comment (justinmk) [inline src/nvim/channel.c]: included address in log message

File: src/nvim/channel.c [hunk 2]
```diff
@@ -447,6 +449,7 @@ static uint64_t channel_create_internal_rpc(void)
 {
   Channel *channel = channel_alloc(kChannelStreamInternal);
   rpc_start(channel);
+  channel_create_event(channel, "RPC-internal");
   return channel->id;
```
Comment (justinmk) [inline src/nvim/channel.c]: Also log this. Unless @bfredl intentionally did not want to log it for some reason?

File: src/nvim/channel.c (hunk 1 above)
Comment (bfredl) [reply src/nvim/channel.c]: I though of adding the address in the channel info, but this is good for now :+1:

File: src/nvim/channel.c (hunk 2 above)
Comment (bfredl) [reply src/nvim/channel.c]: Good catch, omission due to early return in caller. I would rather refactor the caller to `goto end` and make it reuse the same `channel_create_event(channel, address)`.

**Pull request merged**